ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -0,0 +1,188 @@
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+import {withRouteData} from 'app/components/app-router';
+import {Button} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {FlexColumn, } from 'app/components/flex';
+import {Arrow, ClrIcon, withCircleBackground} from 'app/components/icons';
+import {RadioButton} from 'app/components/inputs';
+import {AoU} from 'app/components/text-wrappers';
+import {withProfileErrorModal} from 'app/components/with-error-modal';
+import {styles} from 'app/pages/profile/profile-styles';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {
+  daysFromNow,
+  displayDateWithoutHours,
+  useId,
+  withStyle
+} from 'app/utils';
+import {profileStore, useStore} from 'app/utils/stores';
+
+const {useState} = React;
+const LOOKBACK_PERIOD = 330;
```

Comment:
Should this be derived from the server config value instead? Also, can you add a comment about what this variable represents specifically?

Reply:
Added a ticket to address this